### DISCUSSION TITLE
object-position="top"

### DISCUSSION THREAD

**Discussion by axjms1** [willfaught/paige#36] [Q&A] [answered]
Last question for the day.  

How would someone go about modifying the paige/image shortcode so that it supported the object-position="top" attribute?

**Comment by willfaught** [comment]
Have you tried setting the style parameter?

**Reply by axjms1** [reply]
Hi Will,  Is this a parameter in the paige/image shortcode?

**Reply by willfaught** [reply, marked as answer]
Yes: https://github.com/willfaught/paige#image

**Reply by axjms1** [reply]
sorry I missed that.  I was looking at the shortcode guide here: https://willfaught.com/paige/shortcodes/image/

I will try that thank you.

**Reply by willfaught** [reply]
Thanks for pointing out that the style parameter wasn't shown on that page. I've fixed the documentation to show every shortcode parameter. Please report any other missing things! I found and fixed a bug as part of this work, and I released it as v0.65.3.

**Reply by axjms1** [reply]
Thank you for all your work on this project
